src/dist/normal_inv_chi_squared.rs

```diff
@@ -404,6 +404,8 @@ impl Sampleable<Gaussian> for NormalInvChiSquared {
         let post_sigma: f64 = sigma / self.k.sqrt();
         let mu: f64 = Gaussian::new(self.m, post_sigma)
             .map_err(|err| {
+                dbg!(&self);
+                dbg!(var, sigma, post_sigma);
                 panic!("Invalid μ params when drawing Gaussian: {err}")
```

Comment:
I think it'd be best to add the values into the panic so there's one message with everything we need.